plugins/woocommerce/includes/data-stores/class-wc-order-data-store-cpt.php

```diff
@@ -205,7 +215,7 @@ public function update( &$order ) {
 		$previous_status = OrderUtil::remove_status_prefix( $previous_status );
 		$current_status  = OrderUtil::remove_status_prefix( $current_status );
 
-		$draft_statuses = array( 'new', 'auto-draft', 'draft', 'checkout-draft' );
+		$draft_statuses = array( 'new', OrderStatus::AUTO_DRAFT, OrderStatus::DRAFT, 'checkout-draft' );
```

Comment:
Depending on our decision to the earlier question re `checkout-draft`, we *might* want to update this line.